extension/src/time_vector/pipeline.rs

```diff
@@ -73,14 +61,8 @@ pub mod toolkit_experimental {
             LTTB: 1 {
                 resolution: u64,
             },
-            ResampleToRate: 2 {
-                interval: i64,
-                resample_method: ResampleMethod,
-                snap_to_rate: i64, // padded bool
-            },
-            FillHoles: 3 {
-                fill_method: FillHolesMethod,
-            },
+            // 2 was for resample_to_rate
+            // 3 was for fill_holes
```

Comment:
Note that we can replace these if we so choose; the update script will make sure that the gaps aren't used. (We don't need to now, but we should probably do so for when we add new elements)

Reply:
I think we'll ultimately want to bring back resample and fill_holes once we plan them out better, though of course we can fill these gaps with anything